activation/handler.go

```diff
@@ -112,7 +112,7 @@ func (h *Handler) ProcessAtx(ctx context.Context, atx *types.ActivationTx) error
 	h.processAtxMutex.Lock()
 	defer h.processAtxMutex.Unlock()
 
-	existingATX, _ := h.cdb.GetAtxHeader(atx.ID())
+	existingATX, _ := h.cdb.GetAtxByID(atx.ID())
```

Comment:
do we skip error here on purpose? 🤔

Reply:
We are only checking if the ATX with the given ID already exists in the cache / db. If it does, `existingATX` will be non-nil and `err` will be nil. If the ATX does not exist in the cache and cannot be fetched it will be `nil` and `err` will be non-nil.

The returned error will be some SQL error that would need to be parsed to be handled correctly. The only error that could give extra info besides "atx hasn't been stored yet" are DB connection issues. These arguably aren't needed to be handled here, since they will be handled a few lines further down if storing the ATX fails anyway.